ambry-protocol/src/main/java/com.github.ambry.protocol/TtlUpdateRequest.java

```diff
@@ -0,0 +1,176 @@
+/*
+ * Copyright 2018 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.protocol;
+
+import com.github.ambry.clustermap.ClusterMap;
+import com.github.ambry.commons.BlobId;
+import com.github.ambry.utils.Utils;
+import java.io.DataInputStream;
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.nio.channels.WritableByteChannel;
+
+import static com.github.ambry.protocol.RequestOrResponseType.TtlUpdateRequest;
+
+
+/**
+ * Request to update the TTL of a blob
+ */
+public class TtlUpdateRequest extends RequestOrResponse {
+
+  static final short TTL_UPDATE_REQUEST_VERSION_1 = 1;
+  private final static short CURRENT_VERSION = TTL_UPDATE_REQUEST_VERSION_1;
+
+  private final short version;
```

Comment:
`version` does not serve any purpose in this class as this is duplicated in the super class (where it is a protected variable). Serialization ignores this field for the same reason. This is true for `DeleteRequest` as well. Better get rid of this from both classes.